src/main/resources/default/extensions/tycho-page-menu/footer.html.pasta

```diff
@@ -3,6 +3,16 @@
 
 <i:switch test="@point">
     <i:block name="footer-right-start">
+        <i:if test="!currentRenderContext().getRootContext().getTemplate().getName().endsWith('dashboard.html.pasta')">
```

Comment:
Da gibts keinen besseren Check für? Vllt den Block im Dashboard Template überschreiben oder so?

Reply:
Would probably look as ugly as that. Sadly, we cannot simply extract this into a block, as we're already in an extension...